app/src/main/java/intellistart/interviewplanning/controllers/nats/slot/CreateSlotNatsController.kt

```diff
@@ -0,0 +1,92 @@
+package intellistart.interviewplanning.controllers.nats.slot
+
+import com.google.protobuf.Parser
+import intellistart.interviewplanning.NatsSubject.Slot.CREATE
+import intellistart.interviewplanning.controllers.dto.toProto
+import intellistart.interviewplanning.controllers.nats.NatsController
+import intellistart.interviewplanning.exceptions.SlotException
+import intellistart.interviewplanning.exceptions.UserException
+import intellistart.interviewplanning.model.period.PeriodService
+import intellistart.interviewplanning.model.slot.Slot
+import intellistart.interviewplanning.model.slot.SlotService
+import intellistart.interviewplanning.model.slot.validation.SlotValidator
+import intellistart.interviewplanning.request.slot.create.proto.CreateSlotRequest
+import intellistart.interviewplanning.request.slot.create.proto.CreateSlotResponse
+import io.nats.client.Connection
+import org.bson.types.ObjectId
+import org.springframework.stereotype.Component
+
+@Component
+class CreateSlotNatsController(
+    override val connection: Connection,
```

Comment:
You don't use connection in this class, and you won't use it in the next one either, so think about whether you need it here at all

Reply:
To create a Dispatcher, you have a BPP, think about whether you need to pass a connection to each class.